### DISCUSSION TITLE
Flickering when switching tabs in an organization's shared layout

### DISCUSSION THREAD

**Discussion by Woodii1998** [foxglove/community#972] [Bugs]
**Description**
Occasional fallbacks when having multiple tabs and switching back and forth in an organization's shared layout

- Version: 1.83.1+87877a4.60ca5a8
- Platform: web
- Data source (e.g. bag file, mcap file, rosbridge, ROS 1/2 native): bag file

**Steps To Reproduce**

https://github.com/foxglove/studio/assets/33593572/4686af02-fa13-453b-9728-e2d4ead16d70

1. Open an organization's shared layou
2. Create multiple tab
3. Open a panel with data in each tab.
4. Switch tab 

It looks like there's a timing conflict between updating and pulling the organization's shared layout

**Expected Behavior**
No fallback should occur when switching tabs

**Comment by jtbandes** [comment]
Internal tracking issue: FG-6276

**Comment by defunctzombie** [comment]
This was fixed about two months ago but we forgot to update here. Thanks for reporting.
